app/api_topology.go

```diff
@@ -133,6 +134,8 @@ func handleWebsocket(
 	defer rep.UnWait(ctx, wait)
 
 	for {
+		span := ot.StartSpan("ws.Render", ot.Tag{"topology", topologyID})
```

Comment:
I wonder if we'd also want to add `channelOpenedAt` and `reportTimestamp` to the span tags.

Reply:
Also, `websocket.Render` for more clarity?

Reply:
Tags are indexed for lookup; it's rather unlikely someone would want to do that so I don't feel the overhead is warranted.
I could add them in a `Log` line since the data is interesting.

Reply:
> I could add them in a Log line since the data is interesting.

yeah, maybe you can do that

Reply:
Done.